pandas_gbq/gbq.py

```diff
@@ -409,10 +407,10 @@ def run_query(self, query, **kwargs):
 
         self._start_timer()
         try:
-            self._print('Requesting query... ', end="")
+            logger.info('Requesting query... ', end="")
```

Comment:
Typo here that results in the following exception: `TypeError: _log() got an unexpected keyword argument 'end'`

```
    logger.info('Requesting query... ', end="")
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <logging.Logger object at 0x7fc77b571150>, msg = 'Requesting query... ', args = ()
kwargs = {'end': ''}

    def info(self, msg, *args, **kwargs):
        """
            Log 'msg % args' with severity 'INFO'.
    
            To pass exception information, use the keyword argument exc_info with
            a true value, e.g.
    
            logger.info("Houston, we have a %s", "interesting problem", exc_info=1)
            """
        if self.isEnabledFor(INFO):
>           self._log(INFO, msg, args, **kwargs)
E           TypeError: _log() got an unexpected keyword argument 'end'
```